doc/Paso_a_paso_DIA_1.md

```diff
@@ -0,0 +1,64 @@
+# Paso a Paso: Día 1
+
+A través de esta serie de tutoriales escritos construiremos juntos el videojuego Snake en Python desde cero. No importa tu nivel de experiencia con Python, te llevaremos paso a paso y te explicaremos todo con lujo de detalles cada línea de código del proyecto. Si te parece empezamos ya con el primer día de código.
```

Comment:
It seems better to say:

"Te explicaremos con lujo de detalles". Remove "todo"